src/components/AddPlaidBankAccount.js

```diff
@@ -119,30 +115,30 @@ class AddPlaidBankAccount extends React.Component {
 
     /**
      * Triggered when user selects a Plaid bank account.
-     * @param {String} index
+     * @param {String} plaidAccountID
```

Comment:
The changes here will conflict with https://github.com/Expensify/App/pull/10953/files#diff-2a35d342f7be1cc5c1a8f02937f635a5c7083ec66b64b400027d65087d400e09 👀 

cc: @ctkochan22 

I do like the idea of using the plaidAccountID instead of the index, though.